docs/crio.conf.5.md

```diff
@@ -258,6 +258,7 @@ The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.  Th
   "io.kubernetes.cri-o.userns-mode" for configuring a user namespace for the pod.
   "io.kubernetes.cri-o.Devices" for configuring devices for the pod.
   "io.kubernetes.cri-o.ShmSize" for configuring the size of /dev/shm.
+  "io.kubernetes.cri-o.UnifiedCgroup.$CTR_NAME" for configuring the cgroup v2 unified block for a container.
```

Comment:
why $CTR_NAME? why not have it apply to the whole pod?

Reply:
cgroups are set at the container level in pod yaml

Reply:
.. fair enough :)